test/test_configuration.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2021- Python Language Server Contributors.
+
+import os
+from unittest.mock import patch
+
+from test.fixtures import CALL_TIMEOUT_IN_SECONDS
```

Comment:
```suggestion
from test.test_utils import CALL_TIMEOUT_IN_SECONDS
```

Adjust this for the latest changes in `develop`.